pkg/poller/poller.go

```diff
@@ -125,7 +126,7 @@ func (o *Options) pollRepository(r repo.Repository) error {
 		}
 	} else {
 		if o.Branch == "" {
-			o.Branch, err = gitclient.Branch(o.GitClient, dir)
+			o.Branch, _ = gitclient.Branch(o.GitClient, dir)
```

Comment:
swalloing errrors is never good. I would atleast log it as a warning message.

Reply:
The linter detected it was never used as at line 133, it gets a new value so I thought it's not useful.
you suggest adding this instead?
```
		if err != nil {
			return errors.Wrapf(err, "failed to pull repository %s", name)
		}
```

Reply:
Dont return, just use logger with warning level.

Reply:
done